dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/config/liberty/CDI12ContainerConfig.java

```diff
@@ -48,8 +47,7 @@ protected void activate(ComponentContext compcontext, Map<String, Object> proper
         if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
             Tr.debug(tc, "Activating " + this);
         }
-        //TOOD the ProductInfo.getBetaEdition() check should be removed before GA
-        if (TraceComponent.isAnyTracingEnabled() && tc.isInfoEnabled() && ProductInfo.getBetaEdition()) {
+        if (TraceComponent.isAnyTracingEnabled() && tc.isInfoEnabled()) {
```

Comment:
@tevans78 is on vacation.  He put this check in.  Not sure if this is what was intended or not.  Do we want this message to come out every time or only when the customer enables some sort of fine, finer or finest trace?  If the former the isAnyTracingEnabled should be removed.  @Emily-Jiang or @benjamin-confino do you know about what the conversations where related to this change?

Reply:
I am not sure. @Azquelt do you know anything about this?

Reply:
Good spot @jhanders34, this looks like a bug. The message should be output whether tracing is enabled or not. The intention is thankfully listed [in the UFO](https://ibm.ent.box.com/file/925747728313) (slide 45).

It looks like the test which checks this message doesn't intentionally set any trace specification, but when it runs it has `*=info:com.ibm.ws.runtime.update.*=all:logservice=detail` which I guess is added automatically.

Reply:
Thanks Andrew.  I have updated the code to remove the if statement completely.